app/main/views/services.py

```diff
@@ -113,6 +105,72 @@ def update_service_status(service_id):
     )
 
 
+@main.route(
+    '/services/<string:service_id>/edit/<string:section>',
+    methods=['GET']
+)
+@login_required
+@flask_featureflags.is_active_feature('EDIT_SERVICE_PAGE')
+def edit_section(service_id, section):
+    service_data = data_api_client.get_service(service_id)['services']
```

Comment:
This endpoint needs a check that the service belongs to the currently logged-in supplier.  Otherwise if I'm logged in I can try to edit anyone else's services (the POST will fail, but still it looks bad...)